msg/zh_CN.utf8/utils.msg

```diff
@@ -265,6 +266,18 @@ cubrid 管理工具，版本 %1$s\n\
     stop        [数据库名]\n\
     restart     [数据库名]\n\
     status      [数据库名]\n
+45 cubrid 应用工具, 版本 %1$s\n\
```

Comment:
check also message for "jsp" and modify the messages for "gateway" like jsp's or others.